src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -25,47 +25,57 @@
+        <li><p>On the security warning prompt, select <strong>Open</strong>.</p></li>
+        <li><p>On the Certificate screen, select <strong>Install Certificate</strong>.</p></li>
+        <li><p>On the Certificate Import Wizard, select <strong>Next</strong>.</p></li>
+        <li><p>Select <strong>Place all certificates in the following store</strong>, and then select <strong>Browse</strong>.</p></li>
+        <li><p>On the Select Certificate Store screen, select <strong>Trusted Root Certification Authorities</strong>, and then select <strong>OK</strong>.</p></li>
+        <li><p>On the Certificate Import Wizard, select <strong>Next</strong>.</p></li>
+        <li><p>Select <strong>Finish</strong>.</p></li>
+        <li><p>On the security warning prompt, select <strong>Yes</strong>.</p></li>
+        <li><p>On the Certificate Import Wizard confirmation prompt, select <strong>OK</strong>.</p></li>
+        <li><p>After you return to the Certificate screen, select <strong>OK</strong>.</p></li>
     </ol>
 
     <stache-page-anchor>
         Installation for Firefox
     </stache-page-anchor>
     <ol>
         <li><p>Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>.</p></li>
-        <li><p>Type or copy <strong>about:preferences#privacy</strong> into the Firefox address bar.</p></li>
-        <li><p>Click <strong>View Certificates</strong> (near the bottom).</p></li>
-        <li><p>Click <strong>Authorities</strong>.</p></li>
-        <li><p>Click <strong>Import</strong>.</p></li>
-        <li><p>Open the previously downloaded <strong>skyux-ca.crt</strong>.</p></li>
-        <li><p>Check <strong>Trust this CA to identify websites</strong>.</p></li>
-        <li><p>Click <strong>OK</strong> on the options dialog.</p></li>
-        <li><p>Click <strong>OK</strong> on the certificates dialog.</p></li>
+        <li><p>Enter "about:preferences#privacy" in the Firefox address bar.</p></li>
+        <li><p>Near the bottom of the page, select <strong>View Certificates</strong>.</p></li>
+        <li><p>Under <strong>Authorities</strong> on the Certificate Manager, select <strong>Import</strong>.</p></li>
+        <li><p>Open the <strong>skyux-ca.crt</strong>.</p></li>
+        <li><p>On the options dialog, select <strong>Trust this CA to identify websites</strong> and then select <strong>OK</strong>.</p></li>
+        <li><p>On the Certificate Manager, select <strong>OK</strong>.</p></li>
     </ol>
 
+    <stache-page-anchor>
+        Installation for Linux
+    </stache-page-anchor>
+    <ol>
+        <li><p>Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>.</p></li>
+        <li><p>Move the <stache-code>skyux-ca.crt</stache-code> to <stache-code>/usr/local/share/ca-certificates</stache-code>.</p></li>
+        <li><p>From the command prompt, run <stache-code>sudo update-ca-certificates</stache-code>.</p>
```

Comment:
Does "command prompt" sound like Windows?  I would have though "terminal" for a Linux person, but maybe not?

Reply:
I'll change it to "terminal."